pkg/create/templates/scripts.go

```diff
@@ -51,7 +51,7 @@ To use it, install S2I: https://github.com/openshift/source-to-image
 
 Sample invocation:
 
-s2i build git://<source code> {{.ImageName}} <application image>
+s2i build <source code path/URL> {{.ImageName}} <application image>
```

Comment:
I'm okay either way, we can change it.

For context, this is part of the usage script template, used to bootstrap new S2I builder images. Obviously, it won't affect the comments on any existing image.

If the order is really relevant, in my experience I'd use URLs in OpenShift, but tend to use local paths with the `s2i` tool: I often have a local clone of my repo, so I don't need to have `s2i` fetch it from the Internet all the time.